TCAT/Supporting Files/Keys.swift

```diff
@@ -13,8 +13,10 @@ enum Keys: String {
     case fabricAPIKey = "fabric-api-key"
     case fabricBuildSecret = "fabric-build-secret"
 
-    case googleMaps = "google-maps"
-    case googlePlaces = "google-places"
+    case googleMapsDebug = "google-maps-debug"
+    case googleMapsRelease = "google-maps-release"
+    case googlePlacesDebug = "google-places-debug"
+    case googlePlacesRelease = "google-places-release"
```

Comment:
I believe I deleted `GooglePlaces` entirely from the PR i made yesterday so I think you can just delete these two keys